lms/djangoapps/ccx/tests/test_views.py

```diff
@@ -181,6 +198,131 @@ def test_forbidden_user_access_coach_dashboard(self):
 
 
 @attr('shard_1')
+@override_settings(
+    XBLOCK_FIELD_DATA_WRAPPERS=['lms.djangoapps.courseware.field_overrides:OverrideModulestoreFieldData.wrap'],
+    MODULESTORE_FIELD_OVERRIDE_PROVIDERS=['ccx.overrides.CustomCoursesForEdxOverrideProvider'],
+)
+@patch('xmodule.x_module.XModuleMixin.get_children', patched_get_children, spec=True)
+class TestCCXProgressChanges(CcxTestCase, LoginEnrollmentTestCase):
+    """
+    Tests ccx schedule changes in progress page
+    """
+
+    def setUp(self):
```

Comment:
do you need a new version of course and content for each test or can you reuse it? because if you put everything in a `setUpClass` the tests will run much faster